core/src/validator.rs

```diff
@@ -708,6 +708,9 @@ impl Validator {
         };
 
         let (stats_reporter_sender, stats_reporter_receiver) = channel();
+        // https://github.com/rust-lang/rust/issues/39364#issuecomment-634545136
+        let _stats_reporter_sender = stats_reporter_sender.clone();
```

Comment:
ooft this is an issue that has bitten us multiple times.

For clarity and future reference, what in this change set introduced this panic?

Reply:
Sorry I didn't add a comment. It wasn't introduced by these changes. I included it here, but I merged this fix separately: https://github.com/solana-labs/solana/pull/22227